elegant-progressbar-for-a-more-civilized-ageConfig.cmake

```diff
@@ -0,0 +1,5 @@
+# - Config file for the progressbar-for-a-more-civilized-age package
+cmake_minimum_required(VERSION 3.0.2)
+project(elegant-progressbar-for-a-more-civilized-age)
+
+include_directories("${CMAKE_CURRENT_LIST_DIR}/include")
```

Comment:
this should probably become
`include_directories("${CMAKE_CURRENT_LIST_DIR}/../include")`, if the cmake config is located in its own `cmake` folder